Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -117,6 +122,48 @@ module.exports = class DoubleClickToEdit {
 					}
 					//Discord Is One Of Those
 				}, "Enable Replying"),
+
+				React.createElement(this.FormTitle, {
+					tag: "h3",
+				}, "Key to hold when editing a message"),
+				//Allow the user to control when they edit a message
+				React.createElement(this.RadioItem, {
+					value: editActionKeyState,
+					options: [
+						{ name: "None", value: "none" },
+						{ name: "Shift", value: "shift" },
+						{ name: "Ctrl", value: "ctrl" },
+						{ name: "Any", value: "any" }
+					],
+					onChange: (newState) => {
+						this.editActionKey = newState.value;
+						BdApi.saveData(config.info.id, "editActionKey", newState.value);
+						setEditActionKeyState(newState.value);
+					}
+					//Discord Is One Of Those (i have no clue what that means, but pop off ig)
+				}, "Edit ActionKey"),
+
+				React.createElement(this.FormTitle, {
+					tag: "h3",
+					disabled: !replyState
+				}, "Key to hold when replying to a message"),
+				//Allow the user to control when they reply to a message
+				React.createElement(this.RadioItem, {
+					disabled: !replyState,
+					value: replyActionKeyState,
+					options: [
+						{ name: "None", value: "none" },
+						{ name: "Shift", value: "shift" },
+						{ name: "Ctrl", value: "ctrl" },
+						{ name: "Any", value: "any" }
+					],
+					onChange: (newState) => {
+						this.replyActionKey = newState.value;
+						BdApi.saveData(config.info.id, "replyActionKey", newState.value);
+						setReplyActionKeyState(newState.value);
+					}
+				}, "Reply ActionKey"),
+
 				React.createElement(this.SwitchItem, {
 					//The state that is loaded with the default value
 					value: copyState,
```

Comment:
```suggestion
				//Copy
				React.createElement(this.SwitchItem, {
					value: copy,
					note: "Copy selection before entering edit-mode.",
					onChange: (newState) => {
						this.doubleClickToCopy = newState;
						Data.save(config.info.id, "doubleClickToCopy", newState);
						setCopy(newState);
					}
				}, "Enable Copying"),
				React.createElement(this.FormTitle, {
					disabled: !copy,
					tag: "h3"
				}, "Modifier to hold before copying text"),
				React.createElement(this.RadioItem, {
					disabled: !copy,
					value: copyModifier,
					options: [
						{ name: "Shift", value: "shift" },
						{ name: "Ctrl", value: "ctrl" },
						{ name: "Alt", value: "alt" }
					],
					onChange: (newState) => {
						this.copyModifier = newState.value;
						Data.save(config.info.id, "copyModifier", newState.value);
						setCopyModifier(newState.value);
					}
				})
```
And finally the copy section.